pkg/inspector/image-inspector.go

```diff
@@ -499,27 +526,28 @@ func (i *defaultImageInspector) getAuthConfigs() (*docker.AuthConfigurations, er
 	return imagePullAuths, nil
 }
 
-func (i *defaultImageInspector) scanImage(s iiapi.Scanner) ([]byte, []byte, error) {
+// TODO: Move this into openscap package
```

Comment:
afaik this was just wrapping up openscap to get the report and html report... not every scanner will return these (like clam). guess we don't want to have specialized wrapper but rather honor the interface?